### PR TITLE
pager: check_sig minor improvements

### PR DESCRIPTION
**Pull request by ngortheone**
* **What does this PR do?**

Minor readability and type improvements

### CODE REVIEW

File: pager.c [hunk 1]
```diff
@@ -203,6 +203,10 @@ struct PagerRedrawData
 
 /* hack to return to position when returning from index to same message */
 static int TopLine = 0;
+
+
+/* The amount of lines a signature takes */
+static const unsigned int NUM_SIG_LINES = 4;
```
Comment (gahr) [inline pager.c:209]: Can we make it local to the function that uses it?

Comment (ngortheone) [reply pager.c]: done

**Review by gahr — APPROVED** [review]
LGTM - I am curious on the choice of unsigned int, though. We're probably fine with `int` if you just need an integral type, or `size_t` if you really need a counter.
There's been discussions lately on whether we should use `int` or `size_t` for loop indices and counters that we know cannot be negative. There are pros and cons to each, and people have pretty strong opinions (e.g., only use `unsigned` if you want modulo airthmetics). What's your rationale for `unsigned int` here?

**Comment by ngortheone** [comment]
@gahr `size_t` is probably a good choice. Don't forget that I am a homegrown C hacked with a few days of experience

**Pull request merged**